test/testenv/gitea.go

```diff
@@ -318,6 +334,7 @@ func PreGiteaInstallHook(giteaInput *DeployGiteaInput) {
 	case e2e.ManagementClusterEnvironmentIsolatedKind:
 		giteaInput.ServiceType = corev1.ServiceTypeNodePort
 	case e2e.ManagementClusterEnvironmentKind:
+	case e2e.ManagementClusterEnvironmentInternalKind:
```

Comment:
I will leave it to you to accept or leave it as is, but I thought below is bit more readable :)

```suggestion
	case e2e.ManagementClusterEnvironmentKind, e2e.ManagementClusterEnvironmentInternalKind:
```